packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -106,7 +106,7 @@
         border-radius: $BORDER_RADIUS;
         border: solid 1px var(--border-normal);
         min-height: calc(100vh - 30rem);
-        height: calc(100% - 30rem); // TODO: What is 37px?
+        height: calc(100% - 40rem); // TODO: What is 37px?
```

Comment:
May I know what is the unfinished business here?

Reply:
this was a comment to fix the preview height will remove it